app/src/main/java/com/android/sample/ui/authentication/AddAccount.kt

```diff
@@ -432,13 +496,17 @@ fun <T> DropdownMenuButton(
   var expanded by remember { mutableStateOf(false) }
 
   ExposedDropdownMenuBox(expanded = expanded, onExpandedChange = { expanded = !expanded }) {
-    OutlinedTextField(
+    TextField(
         value = selectedOption.toString(),
         onValueChange = {},
+        textStyle = TextStyle(fontFamily = OpenSans, fontSize = SubtitleFontSize),
         readOnly = true,
-        label = { Text("Unit") },
+        label = { Text("Unit", fontFamily = OpenSans) },
         trailingIcon = { ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded) },
-        modifier = modifier.menuAnchor().clickable { expanded = true })
```

Comment:
Indeed the outlinedTextField was not necessary here as we fill the input text with color.